028-HubAndSpoke/Coach/README.md

```diff
@@ -0,0 +1,28 @@
+# Azure Networking - What The Hack - Info for coaches

```

Comment:
Change this to match the root README: "What The Hack - Azure Networking with Hub and Spoke" (no need to reference coach here)

Reply:
Done